libs/core/src/main/java/de/westnordost/osmapi/common/GpxInputStreamWriter.java

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.osmapi.common;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+
+import de.westnordost.osmapi.ApiRequestWriter;
+
+public class GpxInputStreamWriter implements ApiRequestWriter
+{
+	private static final String CHARSET = "UTF-8";
+
+	private InputStream gpx;
+
+	public GpxInputStreamWriter(InputStream in)
+	{
+		this.gpx = in;
+	}
+
+	@Override
+	public String getContentType()
+	{
+		return "text/plain";
```

Comment:
```suggestion
		return "application/gpx+xml";
```

According to Wikipedia, the most correct content type is `application/gpx+xml`.